spec/request/api/v1/users/create_spec.rb

```diff
@@ -24,20 +30,16 @@
     end
 
     it 'returns a successful response' do
-      post user_registration_path, params: params, as: :json
-
-      expect(response).to have_http_status(:success)
+      expect(subject).to have_http_status(:success)
     end
 
     it 'creates the user' do
       expect do
-        post user_registration_path, params: params, as: :json
+        subject
```

Comment:
you can put this in 1 line like `expect { subject }.to change {User.count}.by(1)`